pika/heartbeat.py

```diff
@@ -8,38 +8,37 @@
 
 
 class HeartbeatChecker(object):
-    """Checks to make sure that our heartbeat is received at the expected
-    intervals.
+    """Sends heartbeats to the broker and checks to make sure that the broker's
+    heartbeat is received before the expected timeout expires.
 
     """
-    DEFAULT_INTERVAL = 60
-    MAX_IDLE_COUNT = 2
-
+    _MAX_IDLE_COUNT = 2
```

Comment:
There is no use for _MAX_IDLE_COUNT any longer since we configure our check interval as `self._check_interval = (float(timeout) * 3) / 2` (i.e.   timeout * 1.5) to account for latencies already. Otherwise, with _MAX_IDLE_COUNT of 2, a negotiated timeout of 10 minutes becomes a 30-minute timeout (10 * 1.5 * 2), a bit much, right? If you're an app developer and you limited timeout to x seconds to detect dead connections in some reasonable time, but pike obliged 3x later, wouldn't you and your SLA be fuming right now? :) So, no more _MAX_IDLE_COUNT.